hosted/src/NodeAllocator.cc

```diff
@@ -96,7 +144,8 @@ void ebbrt::NodeAllocator::Session::Start() {
       }));
 }
 
-ebbrt::NodeAllocator::NodeAllocator() : node_index_(2), allocation_index_(0) {
+ebbrt::NodeAllocator::NodeAllocator()
+    : node_index_(2), allocation_index_(0), cmdline_{std::string()} {
```

Comment:
the constructor for `cmdline_` is superfluous, it will default construct it so long as you don't list it